tests/test_modeling_common.py

```diff
@@ -986,6 +987,98 @@ def test_multigpu_data_parallel_forward(self):
             with torch.no_grad():
                 _ = model(**self._prepare_for_class(inputs_dict, model_class))
 
+    @require_multigpu
+    def test_model_parallelization(self):
+        if not self.test_model_parallel:
+            pass
+
+        import subprocess
+
+        def get_current_gpu_memory_use():
+            run_process = subprocess.Popen('nvidia-smi --query-gpu=memory.used --format=csv,nounits,noheader', shell=True, stdout=subprocess.PIPE)
+
+            memory_usage = run_process.stdout.read().decode('utf-8').strip()
+            per_device_memory = [int(memory) for memory in memory_usage.split('\n')]
+            return per_device_memory
+
+        # Needs a large model to see the difference.
+        config = self.model_tester.get_large_model_config()
```

Comment:
Model parallelism is only really visible for large models, hence why I added the `get_large_model_config()` method.